src/request/import/ImportWords.js

```diff
@@ -241,6 +272,9 @@ class ImportWords {
                     wordsExported: true,
                 };
                 this._keyResults.push(result);
+
+                const secretString = Nimiq.BufferUtils.toBase64(key.secret.serialize());
+                sessionStorage.setItem(ImportApi.SESSION_STORAGE_KEY_PREFIX + result.keyId, secretString);
```

Comment:
The IFrameApi does not check the contents of the stored sessionStorage item, only if it exists. Thus the secret does not need to be written into it, it is enough to write anything into to item, like `'This is not the key you are looking for.'`.